orttraining/orttraining/python/training/ortmodule/_torch_module_interface.py

```diff
@@ -0,0 +1,110 @@
+
+    def apply(self: T, fn: Callable[['Module'], None]) -> T:
+
+        raise NotImplementedError(f"apply is not implemented for {type(self)}.")
+
+    def is_training(self):
+
+        raise NotImplementedError(f"is_training is not implemented for {type(self)}.")
+
+    def train(self: T, mode: bool = True) -> T:
+
+        raise NotImplementedError(f"train is not implemented for {type(self)}.")
+
+    def state_dict(self, destination=None, prefix='', keep_vars=False):
+
+        raise NotImplementedError(f"state_dict is not implemented for {type(self)}.")
+
+    def load_state_dict(self, state_dict: 'OrderedDict[str, Tensor]',
+                        strict: bool = True):
+
+        raise NotImplementedError(f"load_state_dict is not implemented for {type(self)}.")
+
+    def register_buffer(self, name: str, tensor: Optional[torch.Tensor], persistent: bool = True) -> None:
+
+        raise NotImplementedError(f"register_buffer is not implemented for {type(self)}.")
+
+    def register_parameter(self, name: str, param: Optional[torch.nn.Parameter]) -> None:
+
+        raise NotImplementedError(f"register_parameter is not implemented for {type(self)}.")
+
+    def get_parameter(self, target: str) -> torch.nn.Parameter:
+
+        raise NotImplementedError(f"get_parameter is not implemented for {type(self)}.")
+
+    def get_buffer(self, target: str) -> torch.Tensor:
+
+        raise NotImplementedError(f"get_buffer is not implemented for {type(self)}.")
+
+    def parameters(self, recurse: bool = True) -> Iterator[torch.nn.Parameter]:
+
+        raise NotImplementedError(f"parameters is not implemented for {type(self)}.")
+
+    def named_parameters(self, prefix: str = '', recurse: bool = True) -> Iterator[Tuple[str, torch.nn.Parameter]]:
+
+        raise NotImplementedError(f"named_parameters is not implemented for {type(self)}.")
+
+    def buffers(self, recurse: bool = True) -> Iterator[torch.Tensor]:
+
+        raise NotImplementedError(f"buffers is not implemented for {type(self)}.")
+
+    def named_buffers(self, prefix: str = '', recurse: bool = True) -> Iterator[Tuple[str, torch.Tensor]]:
+
+        raise NotImplementedError(f"named_buffers is not implemented for {type(self)}.")
+
+    def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
+                              missing_keys, unexpected_keys, error_msgs):
+
+        raise NotImplementedError(f"_load_from_state_dict is not implemented for {type(self)}.")
+
+    def named_children(self) -> Iterator[Tuple[str, 'Module']]:
+
+        raise NotImplementedError(f"named_children is not implemented for {type(self)}.")
+
+    def modules(self) -> Iterator['Module']:
+
+        raise NotImplementedError(f"modules is not implemented for {type(self)}.")
+
+    def named_modules(self, *args, **kwargs):
+
+        raise NotImplementedError(f"named_modules is not implemented for {type(self)}.")
```

Comment:
```suggestion
        raise NotImplementedError(f"named_modules is not implemented for {type(self)}.")

```
NIT: new line

Reply:
Same here, I think there is already a new line at the end of the file.